src/index.html

```diff
@@ -1,14 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div id="qa-block" class="block">
+      <div class="red-border">
+        <div class="blue-border">
+          <div class="red-border">
+            <div class="yellow-border">
+              <div class="red-border">
+                <div class="text">
+                  In programming languages, a closure is a technique for implementing lexically
+                  scoped name binding in a language with first-class functions. Operationally, a
+                  closure is a record storing a function together with an environment.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
You are creating too many unnecessary elements.
If necessary, you can assign multiple classes to one HTML element:
`class="block blue-border"`

Also, you can add red border as border to element and blue or yellow colof as background.
Try completing this task with no more than three `div`s layers.